backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+
+spring.datasource.url=jdbc:mysql://localhost:3306/voyage
+spring.datasource.username=root
+spring.datasource.password=
```

Comment:
are you importing this value from somewhere?

Reply:
those are mySQL credentials for your local machine,